kind/README.md

```diff
@@ -25,10 +26,10 @@ Run the following command to create the cluster:
 
 ```shell
 cd /tmp
-git clone git@github.com:IBM/data-prep-lab.git
-cd data-prep-lab
+git clone git@github.ibm.com:ai-models-data/fm-data-engineering.git
```

Comment:
should it be git@github.com/IBM/data-prep-lab.git
cd data-prep-lab

Why did you change it?